CHANGELOG.md

```diff
@@ -6,8 +6,11 @@
 * System
   * Add version number to config
   * Load arbitrary extra fields into /info from file
-  * Automatically mount usb storage devices as media drives in LMS
-  * Upgrade LMS to 8.5.1
+  * LMS
```

Comment:
We don't have an example of a subheader / subsection, it does seem like we prefer to make new headers rather than having subheaders within what would otherwise be the relevant section
Change made in upcoming commit